src/kudu/tserver/simple_tablet_manager.cc

```diff
@@ -166,7 +166,7 @@ Status TSTabletManager::Load(FsManager *fs_manager) {
     }
   }
 
-  return SetupRaft(server_->opts().edcb);
+  return SetupRaft(server_->opts().edcb, server_->opts().tacb);
```

Comment:
if you use a SetEDCB()|SetTACB() approach, most of these functions should not have to 
grow over time as we add more features.

Reply:
I got the opposite idea from your comments on PR#14. I have used friend classes like we discussed.